test/unit/url.test.js

```diff
@@ -806,7 +811,7 @@ describe('Url', function () {
     });
 
     describe('Security', function () {
-        describe('ReDOS', function () {
+        travisNodeV4Skip('ReDoS', function () {
```

Comment:
Need to add a comment here explaining why it was skipped

Reply:
Also, this isn't going to be used anywhere else, so you can use a literal expression on line 814.